fl4health/clients/evaluate_client.py

```diff
@@ -190,33 +202,12 @@ def get_parameter_exchanger(self, config: Config) -> ParameterExchanger:
         """
         return FullParameterExchanger()
 
-    def predict(self, input: torch.Tensor) -> torch.Tensor:
-        """
-        Return predictions when given input. User can override for more complex logic.
-        """
-        return self.model(input)
-
-    def compute_loss(self, preds: torch.Tensor, target: torch.Tensor) -> Losses:
-        """
-        Computes loss given preds and torch and the user defined criterion. Optionally includes dictionary of
-        loss components if you wish to train the total loss as well as sub losses if they exist.
-        """
-        loss = self.criterion(preds, target)
-        losses = Losses(checkpoint=loss, backward=loss)
-        return losses
-
-    def get_data_loader(self, config: Config) -> DataLoader:
+    def get_data_loader(self, config: Config) -> DataLoader:  # type: ignore
```

Comment:
Why is the type: ignore required here?

Reply:
Wasn't a problem when we extended NumpyFlClient but now it is

Reply:
Yeah I see now. Maybe this is a dumb idea but we could have `get_data_loader` return a type `Tuple[DataLoader, ...]` This allows for a tuple of unspecified length? I'm not sure if it would completely break everything though 😂